give/src/components/Carousel/Carousel.js

```diff
@@ -10,57 +16,66 @@ export const Carousel = ({
 }) => {
   if (content.length === 0) return null;
 
-  const classes = className !== "" ? " " + className : className;
-  navButtonClasses =
-    navButtonClasses !== "" ? " " + navButtonClasses : navButtonClasses;
-
-  return (
-    <div className={"carousel cru-carousel" + navButtonClasses + classes}>
-      <div className="cmp-carousel">
-        <div className="cmp-carousel__content">
-          {content.map((slide, index) => {
-            const active = slide.active ? " cmp-carousel__item--active" : "";
+  const Actions = () => {
+    const actions = [
+      { label: "Previous", classes: "previous" },
+      { label: "Next", classes: "next" },
+      { label: "Pause", classes: "pause" },
+      { label: "Play", classes: "play cmp-carousel__action--disabled" },
+    ];
 
-            return (
-              <div className={"cmp-carousel__item" + active} key={index}>
-                {slide.slide}
-              </div>
-            );
-          })}
-          <div className="cmp-carousel__actions">
-            <button className="cmp-carousel__action cmp-carousel__action--previous">
-              <span className="cmp-carousel__action-icon"></span>
-              <span className="cmp-carousel__action-text">Previous</span>
-            </button>
-            <button className="cmp-carousel__action cmp-carousel__action--next">
-              <span className="cmp-carousel__action-icon"></span>
-              <span className="cmp-carousel__action-text">Next</span>
-            </button>
-            <button className="cmp-carousel__action cmp-carousel__action--pause">
+    return (
+      <div className="cmp-carousel__actions">
+        {actions.map((action, index) => {
+          return (
+            <button
+              className={`cmp-carousel__action cmp-carousel__action--${action.classes}`}
+              key={index}
+            >
               <span className="cmp-carousel__action-icon"></span>
-              <span className="cmp-carousel__action-text">Pause</span>
+              <span className="cmp-carousel__action-text">${action.label}</span>
```

Comment:
Wouldn't this just be {action.label} not ${} since it isn't in a template literal?

Reply:
Good catch.